repose-aggregator/components/filters/valkyrie-authorization/src/main/scala/org/openrepose/filters/valkyrieauthorization/ValkyrieAuthorizationFilter.scala

```diff
@@ -96,12 +117,31 @@ class ValkyrieAuthorizationFilter @Inject()(configurationService: ConfigurationS
       case ResponseResult(code, message) =>
         mutableHttpResponse.sendError(code, message)
     }
+
+    //weep for what we must do with this flaming pile, note above where it was wrapped pointlessly just to break the chain
+    mutableHttpResponse.writeHeadersToResponse()
+    mutableHttpResponse.commitBufferToServletOutputStream()
   }
 
   def cacheKey(transformedTenant: String, contactId: String): String = {
     transformedTenant + contactId
   }
 
+  def nonAuthorizedPath(url: => String): Boolean = {
+    val path: String = new URL(url).getPath
+    lazy val onResourceList: Boolean = Option(configuration.getCollectionResources)
```

Comment:
Any reason this was made lazy? Just to be consistent with the other?

In general, you don't want to use lazy vals unless you need to. A good explanation on why can be found here:
http://stackoverflow.com/a/17328731